frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx

```diff
@@ -62,7 +62,7 @@ export default class PartialQueryBuilder extends Component {
                 }
             }
         };
-        let previewUrl = "/q/" + serializeCardForUrl(previewCard);
+        let previewUrl = "/q#" + serializeCardForUrl(previewCard);
```

Comment:
We switched from /q/... to /q#... awhile ago (#1758). There used to be a redirect for the legacy URLs so this wasn't totally broken, but I think it was removed at some point in the transition from Angular, so it was broken. This is the correct fix.